tech.keyforge.master.yml

```diff
@@ -0,0 +1,50 @@
+app-id: "tech.keyforge.master"
+runtime: "org.freedesktop.Platform"
+runtime-version: "23.08"
+sdk: "org.freedesktop.Sdk"
+sdk-extensions:
+  - "org.freedesktop.Sdk.Extension.node20"
+base: "org.electronjs.Electron2.BaseApp"
+base-version: "23.08"
+command: run.sh
+finish-args:
+  - "--device=all"
+  - "--share=ipc"
+  - "--share=network"
+  - "--socket=x11"
+
+modules:
+  - name: "keyforge-master"
+    buildsystem: "simple"
+    build-commands:
+      - mkdir /app/main
+      - tar -xf keyforge-master_0.5_linux.tar.xz -C /app/main --strip-components=1
+      - rm  keyforge-master_0.5_linux.tar.xz
+      - install -Dm 644 tech.keyforge.master.desktop /app/share/applications/tech.keyforge.master.desktop
+      - install -Dm 644 tech.keyforge.master.metainfo.xml /app/share/metainfo/tech.keyforge.master.metainfo.xml
+      - install -Dm 755 run.sh /app/bin/run.sh
+      - install -Dm 644 64x64.png /app/share/icons/hicolor/64x64/apps/tech.keyforge.master.png
+      - install -Dm 644 256x256.png /app/share/icons/hicolor/256x256/apps/tech.keyforge.master.png
+      - install -Dm 644 512x512.png /app/share/icons/hicolor/512x512/apps/tech.keyforge.master.png
+      - install -Dm 644 scalable.svg /app/share/icons/hicolor/scalable/apps/tech.keyforge.master.svg
```

Comment:
Just the scalable icon is fine, you don't need the PNGs.

Reply:
I have removed the PNGs, thanks.